grails-app/assets/stylesheets/elurikkus-common.css

```diff
@@ -766,3 +766,39 @@ section {
     display: inline-block;
     margin: 0 .25rem;
 }
+
+/* Gallery styling */
+.imgCon {
+    display: inline-block;
+    text-align: center;
+    line-height: 1.3em;
+    background-color: #DDD;
+    color: #DDD;
+    font-size: 12px;
+    margin: 2px 4px 2px 0;
+    position: relative;
+}
+
+.imgCon img {
+    height: 170px;
+    min-width: 150px;
+}
+
+.imgCon .brief {
+    opacity: 0.8;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    overflow: hidden;
+    text-align: left;
+    padding: 4px 5px 2px 5px;
+    color: black;
+    background-color: white;
+}
+
+.imgCon .detail {
```

Comment:
Are these classes set by our code or some plugin? If possible, styles should follow BEM conventions to prevent specificity problems:

```css
.gallery-container {
   ...
}

.gallery-container__image {
   ...
}

.gallery-container__brief {
   ...
}
```

If img/brief are out of our reach, then at least might rename imgCon